src/components/timeout-dialog/timeout-dialog.js

```diff
@@ -198,19 +236,17 @@ function TimeoutDialog ($module) {
       if (counter <= 0) {
         signOut()
       }
+      currentTimer = window.setTimeout(runUpdate, counter > 60 ? 60000 : 1000)
```

Comment:
I think this condition needs to be counter > 120 or just set the timeout for every second. Otherwise if timeout is between 60 and 120 you are not going to get a countdown until only a few seconds are remaining.